docroot/themes/humsci/humsci_basic/src/scss/components/_media.scss

```diff
@@ -17,3 +17,16 @@
   min-height: hb-calculate-rems(300px);
   margin-bottom: hb-spacing-width();
 }
+
+// Hotfix for caption/credit positioning bug with the media image field
+.field-hs-event-image,
+.field-hs-news-image,
+.field-hs-event-series-image,
+.field-hs-person-square-img,
+.field-hs-publication-image,
+.field-hs-accordion-image, //This still goes full width of the accordion, but keeps it contained to the image area.
+.field-hs-spotlight-image,
+.field-hs-hero-image,
+.custm-new-project-image {
```

Comment:
custm fields are not part of the code base. it's might require an asset injector if that is an issue.

Reply:
I will remove that line, we are supplying an asset injector workaround to Alex as well so that shouldn't be an issue.